common/src/main/java/org/jboss/hal/testsuite/page/config/ServerConfigurationPage.java

```diff
@@ -60,4 +45,18 @@ public ConfigFragment getWindowFragment() {
         WebElement editPanel = browser.findElement(By.className("default-window-content"));
         return  Graphene.createPageFragment(ConfigFragment.class, editPanel);
     }
+
+    public void navigate() {
+        browser.navigate().refresh();
+        Graphene.goTo(HomePage.class);
```

Comment:
I looked into it and it looks like there is only one usage of ServerConfigurationPage - in ELResolverTestCase. Due to the placement of Expression Resolver tool, which is placed in footer and thus available from every location, I just chose the shortest navigation possible.
IMO it would be better solution to just refactor name of this class. @rhatlapa WDYT?